people/praveen-kumar.md

```diff
@@ -0,0 +1,6 @@
+Praveen Kumar
+------------------
+
+![](photos/praveen-kumar.png)
+
+* www.praveenkumar.co.uk
```

Comment:
This should be a Markdown link, so `[Personal Website](http://www.praveenkumar.co.uk)` for example